app/config/tstvhq/rooms.json

```diff
@@ -382,19 +381,15 @@
         {
             "id": "tstHQ",
             "name": "Infernal Theater",
-            "type": "IFRAME",
-            "capacity": 100,
+            "type": "JITSI",
+            "capacity": 20,
             "muteRoom": false,
             "hideVideo": false,
             "locked": false,
             "description": "The crushed velvet seats recline! Are they emerald or black? Hard to tell in the dark.",
-            "iframeOptions": {
-                "src": "https://player.twitch.tv/?channel=LEDflashing&parent=estate.tstvhq.com"
-            },
             "directions": {
                 "west": "staircase",
-                "north": "belfry"
-            },
```

Comment:
this brace and comma shouldn't be deleted

Reply:
'k